digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -295,7 +308,13 @@ protected TaskResult callExecutor(Path projectPath, String type, TaskRequest mer
         DefaultSecretProvider secretProvider = new DefaultSecretProvider(
                 secretContext, secretAccessPolicy, grants, operatorSecretFilter, secretStore);
 
-        TaskExecutionContext taskExecutionContext = new DefaultTaskExecutionContext(secretProvider);
+        PrivilegedVariables privilegedVariables = GrantedPrivilegedVariables.build(
+                mergedRequest.getLocalConfig().getNestedOrGetEmpty("_env"),
```

Comment:
Maybe, yes. But in current implementation, Operator has total control of starting a new process. To set "_env" to a new process, "_env" needs to go through Operator. And because OperatorManager doesn't know whether the Operator starts a new process or not, OperatorManager needs to pass "_env" to Operator every time. Because of this implementation limitation, _env is passed to Operator every time. Do you think we should change this? Then, an idea is to add `@com.google.common.annotations.Beta` annotation to `OperatorContext.getPrivilegedVariables()`.